File: 2st/Yeonwoo/옷가게 할인 받기.py
def solution(price):
    answer = 0
    if price >= 100000 and price < 300000:
        answer = int(price *0.95)
    elif price >= 300000 and price < 500000:
        answer = int(price * 0.9)
    elif price >= 500000:
        answer = int(price * 0.8)
    else:
        answer = int(price)
    return answer

File: 4st/Jisoo/programmer/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        // int answer = 0;
        if(price>=500000){
            return(int)(price*0.8);
        }
        else if(price>=300000){
            return(int)(price*0.9);
        }
        else if (price>=100000){
            return(int)(price*0.95);
        }
        else{
             return price;
        }
    }
}